### PR TITLE
done

### PR DESCRIPTION
**Pull request by gymmish**

### CODE REVIEW

File: controllers/contacts/addContacts.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const { Contact } = require("../../models/contsct");
+
+const addContscts = async (req, res, next) => {
```
Comment (ceREAL133) [inline controllers/contacts/addContacts.js]: addContacts) there's a typo here

File: controllers/contacts/getContact.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+const { Contact } = require("../../models/contsct");
```
Comment (ceREAL133) [inline controllers/contacts/getContact.js]: /../models/contsct "contact" another typo

File: controllers/contacts/index.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+const getListContact = require("./getListContact");
+const getContact = require("./getContact");
+const addContacts = require("./addContacts");
+const removeContact = require("./removeContact");
+const updateContact = require("./updateContact");
+const updateStatus = require("./updateStatus");
+
+module.exports = {
```
Comment (ceREAL133) [inline controllers/contacts/index.js:7]: all those controllers you could just store in one file called contact.controller.js. No need to separate  them for different files

File: models/contsct.js
Comment (ceREAL133) [inline models/contsct.js:1]: file contact.js another typo)

**Comment by gymmish** [comment]
thank you! I fixed everything. It's hard to be careful in Node.js
The terminal did not show those errors.

**Review by ceREAL133 — APPROVED** [review]

**Pull request merged**